docs/integrations/sources/postgres.md

```diff
@@ -1,263 +1,273 @@
+```
+```
+az postgres server restart --resource-group group --name server
 ```
 
-If you would like to use `wal2json` plugin, please change `pgoutput` to `wal2json` value in the above query.
-
-#### <a name="publications-replication"></a>5. Create publications and replication identities for tables
-
-For each table you want to replicate with CDC, you should add the replication identity \(the method of distinguishing between rows\) first. We recommend using `ALTER TABLE tbl1 REPLICA IDENTITY DEFAULT;` to use primary keys to distinguish between rows. After setting the replication identity, you will need to run `CREATE PUBLICATION airbyte_publication FOR TABLE <tbl1, tbl2, tbl3>;`. This publication name is customizable. Please refer to the [Postgres docs](https://www.postgresql.org/docs/10/sql-alterpublication.html) if you need to add or remove tables from your publication in the future.
-
-Please note that:
-- You must **add the replication identity before creating the publication**. Otherwise, `ALTER`/`UPDATE`/`DELETE` statements may fail if Postgres cannot determine how to uniquely identify rows.
-- The publication should **include all the tables and only the tables that need to be synced**. Otherwise, data from these tables may not be replicated correctly.
-
-The UI will only show tables included in the publication. If a table is part of the publication but does not have a replication identity, that replication identity will be created automatically on the first run if the Airbyte user has the necessary permissions.
+#### Step 2: Select a replication plugin​
 
-#### 5. Create replication slot
+We recommend using a [pgoutput](https://www.postgresql.org/docs/9.6/logicaldecoding-output-plugin.html) plugin (the standard logical decoding plugin in Postgres). If the replication table contains multiple JSON blobs and the table size exceeds 1 GB, we recommend using a [wal2json](https://github.com/eulerto/wal2json) instead. Note that wal2json may require additional installation for Bare Metal, VMs (EC2/GCE/etc), Docker, etc. For more information read the [wal2json documentation](https://github.com/eulerto/wal2json).
 
-Next, you will need to create a replication slot. It's important to create the publication first (as in step 4) before creating the replication slot. Otherwise, you can run into exceptions if there is any update to the database between the creation of the two.
+#### Step 3: Create replication slot​
 
-Here is the query used to create a replication slot called `airbyte_slot`:
+To create a replication slot called `airbyte_slot` using pgoutput, run:
 
-```text
+```
 SELECT pg_create_logical_replication_slot('airbyte_slot', 'pgoutput');
 ```
 
-If you would like to use `wal2json` plugin, please change `pgoutput` to `wal2json` value in the above query.
-
-#### <a name="initial-waiting-time"></a>6. [Optional] Set up initial waiting time
```

Comment:
@Amruta-Ranade, why is this entire section deleted? This information is important for the connector set up. In addition, the current set up page has a link to this section, which is broken now.

<img width="608" alt="Screen Shot 2022-08-03 at 16 01 34" src="https://user-images.githubusercontent.com/1933157/182726987-7aec7948-16bb-4838-9f47-32e5e4f780ca.png">

Reply:
It isn't deleted..just restructured. If I'm not mistaken, the previous content didn't explicitly state the command to run. It does now: https://docs.airbyte.com/integrations/sources/postgres#step-3-create-replication-slot

Reply:
I am referring to `6. [Optional] Set up initial waiting time`, not `step 3`.

The doc about initial waiting time is no longer there in the latest master branch:
https://github.com/airbytehq/airbyte/blob/master/docs/integrations/sources/postgres.md

Reply:
Thank you~~

Reply:
PR: https://github.com/airbytehq/airbyte/pull/15297